data/actionsmap/yunohost.yml

```diff
@@ -302,6 +302,14 @@ domain:
             arguments:
                 domain:
                     help: Target domain
+                -t:
+                    full: --ttl
+                    help: Time To Live (TTL) in second before DNS servers update. Default is 3600 second (i.e. 1 hour).
+                    extra:
+                        pattern: &pattern_ttl
+                            - !!str ^[0-9]+$
+                            - "pattern_ttl"
```

Comment:
This string corresponds to the translation key of the message to display in case of the argument doesn't match the pattern. You should consider to define the corresponding English message for that key inside `locales/en.json` - and English only, other languages are managed by Weblate. I know, this is not documented except inside the *moulinette* source code...